FirebaseAppDistribution/Sources/Public/FirebaseAppDistribution.h

```diff
@@ -1,18 +1,15 @@
-/*
- * Copyright 2020 Google
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 
```

Comment:
You'll also need to add `FIRAppDistributionRelease.h` here since it's a new public header.

Reply:
Moved the include up

Reply:
I added an include for FIRAppDistributionRelease.h here, but it is also included via the FIRAppDistribution include. I don't exactly understand why it's needed here.

Reply:
By explicitly adding it here we're not relying on a transitive includes - if the `FIRAppDistribution.h` file switched to use a class forward it could have adverse affects. It's probably best practice to explicitly declare all your public headers for the umbrella header

Reply:
Thanks for the explanation!